src/libstore/filetransfer.cc

```diff
@@ -580,7 +580,12 @@ struct curlFileTransfer : public FileTransfer
         #endif
 
         #if __linux__
-        unshareFilesystem();
+        try {
+            tryUnshareFilesystem();
```

Comment:
Shouldn't the try/catch be moved inside `tryUnshareFilesystem()` since it claims to be best-effort?

Reply:
Maybe it should never throw an exception? Are there any errors that should not be ignored?

Reply:
I am not sure, but I am generally uncomfortable with ignoring errors without a whitelist.